src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,104 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int GROW_COEFFICIENT = 2;
+    private Node<K, V>[] elements;
+    private int threshold;
```

Comment:
threshold variable is unnecessary, please remove it and use other variables values to calculate if resize is required